src/ApplyOfflineMap.cpp

```diff
@@ -282,6 +311,42 @@ try {
 		AnnounceStartBlock("Applying first offline map to data");
 	}
 */
+	
+		DataArray3D<int> dataGLLNodesIn;
+		DataArray3D<double> dataGLLJacobianIn;
+		
+		DataArray3D<int> dataGLLNodesOut;
+		DataArray3D<double> dataGLLJacobianOut;
+		Mesh OverlapMesh;
+		Mesh SourceMesh;
+			
+		//FV or FE input mesh
+		if(strInputMesh != ""){
+			Mesh meshSource(strInputMesh);
+			SourceMesh = meshSource;
+			SourceMesh.ConstructReverseNodeArray();
+			SourceMesh.ConstructEdgeMap();
+			//SourceMesh.RemoveZeroEdges();
+			if(fGLL){
+				double dTotalAreaInput = meshSource.CalculateFaceAreas(fContainsConcaveFaces);
+				double dNumericalAreaIn = GenerateMetaData(meshSource,nPin,false,dataGLLNodesIn,dataGLLJacobianIn);
+			}
+		}
+	
+		//FE output mesh
+		if(strOutputMesh != ""){
+			Mesh meshTarget(strOutputMesh);
+			meshTarget.RemoveZeroEdges();
+			double dTotalAreaOutput = meshTarget.CalculateFaceAreas(fContainsConcaveFaces);
+			double dNumericalAreaOut = GenerateMetaData(meshTarget,nPout,false,dataGLLNodesOut,dataGLLJacobianOut);
+		}
+	
+		if(strOverlapMesh != ""){
+			Mesh meshOverlap(strOverlapMesh);
+			OverlapMesh=meshOverlap;
```

Comment:
This is unnecessary. `meshOverlap` will be created and destroyed within this scope.